libs/designer-ui/src/lib/panel/panelpivot.tsx

```diff
@@ -67,6 +67,19 @@ export const PanelPivot = ({ isCollapsed, tabs, selectedTab, onTabChange, nodeId
 };
 
 function customRenderer(
+  link?: IPivotItemProps,
+  defaultRenderer?: (link?: IPivotItemProps) => JSX.Element | null
+  // isInverted?: boolean
+): JSX.Element | null {
+  // const themeName = isInverted ? 'dark' : 'light';
+  if (!link || !defaultRenderer) {
+    return null;
+  }
+  const navTabDataAutomationId = `msla-panel-pivot-item-${link.headerText?.replace(/\W/g, '_').toLowerCase()}`;
+  return <div data-automation-id={navTabDataAutomationId}>{defaultRenderer({ ...link, itemIcon: undefined })}</div>;
+}
```

Comment:
Is there no way to add extra attributes to the fluent UI element then use the customRenderer?

With commented out code, it should not get checked in.

Reply:
Removing commented-out code.

About modifying FluentUI elements:

I already tried to add extra attributes to the PivotItem FluentUI element, but it did not work. After showing this to my colleagues on Power Automate's Maker Productivity team, they agreed that this was an appropriate solution here.